### PR TITLE
Clean up extra GOLDEN_ANGLES

### LINKED ISSUE #177: Add make_cart_disc and other missing methods to k-wave-array imports and clean up repeated global golden ratios.

### PR DESCRIPTION
**Pull request by waltsims**
closes #177

### CODE REVIEW

File: kwave/utils/mapgen.py [hunk 1]
```diff
@@ -182,7 +181,6 @@ def make_cart_bowl(bowl_pos: np.ndarray, radius: float, diameter: float, focus_p
         # return bowl
         raise NotImplemented("make_cart_disc")
 
-    # compute arc angle from chord (ref: https://en.wikipedia.org/wiki/Chord_(geometry))
```
Comment (faridyagubbayli) [inline kwave/utils/mapgen.py:185]: Why did you delete the comment line? Isn't it useful?

**Review by faridyagubbayli — APPROVED** [review]
LGTM!

File: kwave/utils/mapgen.py (hunk 1 above)
Comment (waltsims) [reply kwave/utils/mapgen.py]: you're right!
